specs/market-api.yaml

```diff
@@ -985,6 +1014,9 @@ components:
         providerId:
           type: string
           readOnly: true
+      required:
+        - offerId
+        - providerId
```

Comment:
ditto

Reply:
That's fine, DemandOfferBase can be used as schema for requestBodies/Demand and requestBodies/Offer.